play-java-compile-di-example/build.sbt

```diff
@@ -2,22 +2,13 @@ name := """play-java-compile-di-example"""
 
 version := "1.0-SNAPSHOT"
 
-lazy val scala213 = "2.13.10"
-lazy val scala3 = "3.3.0-RC3"
-lazy val supportedScalaVersion = List(scala213, scala3)
+lazy val root = (project in file(".")).enablePlugins(PlayJava)
 
-lazy val root = (project in file(".")).enablePlugins(PlayJava).settings(
-    crossScalaVersions := supportedScalaVersion
-  )
+crossScalaVersions := Seq("2.13.10", "3.3.0-RC3")
 
-scalaVersion := scala3
+scalaVersion := crossScalaVersions.value.head
 
-ThisBuild / scalacOptions ++= {
-      CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((3, n))   =>  List("-encoding", "utf8", "-feature", "-unchecked")
-        case _              =>  List("-encoding", "utf8", "-deprecation", "-feature", "-unchecked") 
-      }
-    }
```

Comment:
I now you did that because Scala 3 was complaining about duplicate flags. `-deprecation` does get by Play automatically, so we can just remove it from here. Seems no one noticed so far since Scala 2 never complained.